exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+    "Notes:\n",
+    " - running the following command multiple times will iteratively append the packages\n",
+    " - you can also click on the link and modify the file directly\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe3db39f-5588-44a9-9e64-d4ec495423ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xgboost_pkg = slct_manager.PipPackageDefinition(pkg=\"xgboost\", version=\"2.0.3\")\n",
+    "scikit_learn_pkg = slct_manager.PipPackageDefinition(pkg=\"scikit-learn\", version=\"1.5.0\")\n",
+    "slctmanager.append_custom_packages([xgboost_pkg, scikit_learn_pkg])\n",
+    "show_files([slctmanager.custom_pip_file])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "834a2314-13ab-4ae8-bca5-9d1861376419",
+   "metadata": {},
+   "source": [
+    "#### Rebuilding the customized Flavor\n",
+    "After changing the flavor you need to rebuild it. You can do it by running `export` again. Exaslct automatically recognizes that the flavor has changed and builds a new version of the container."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e7e9bbb-bf77-474f-949f-9521e9e3fd86",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slctmanager.export()\n",
+    "print(\"Ready\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0cec469-33de-4b1a-b481-735e69722c9f",
+   "metadata": {},
+   "source": [
+    "Note: Your old container doesn't get lost, because when you change a flavor your container gets a new hash code. If you revert your changes the system automatically uses the existing cached container. Below you can see the content of the cache directory for the containers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f683dec3-101d-451d-b590-0518a8fdfdc9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "show_directory_content(slctmanager.working_path.output_path / \"cache\" / \"exports\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "67f605e5-0fc6-4e9c-bf6a-1932d04ebdcc",
+   "metadata": {},
+   "source": [
+    "### Testing the new Script-Language Container\n",
+    "\n",
+    "Now, that we have an updated container, we need to check if our changes were successful. For that we are going to upload the container to an Exasol Database and have a look into it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a44b4189-4842-41ce-85e7-f47d4c6ddcec",
+   "metadata": {},
+   "source": [
+    "#### Upload the Container to the Database\n",
+    "To use our container we need to upload it to the BucketFS . If the build machine has access to the BucketFS we can do it with the exaslct upload command, otherwise you need to export the container and transfer it to a machine that has access to the BucketFS and upload it via curl, as described in our documentation.\n",
```

Comment:
```suggestion
    "To use our container we need to upload it to the datebase's so-called _bucket file system_ (BucketFS). If the build machine has access to the BucketFS we can use the command `exaslct upload`, otherwise you need to export the container and transfer it to a machine that has access to the BucketFS and upload it via `curl`, as described in the Exasol documentation.\n",
```

(Maybe add a link to the Exasol documentation online).